protocols/imap/src/main/java/org/apache/james/imap/processor/IdleProcessor.java

```diff
@@ -42,6 +42,7 @@
 import org.apache.james.imap.api.process.SelectedMailbox;
 import org.apache.james.imap.message.request.IdleRequest;
 import org.apache.james.imap.message.response.ContinuationResponse;
+import org.apache.james.mailbox.Event;
```

Comment:
Stange lonely import ?

Reply:
it's not lonely (see line 192 of this file in this commit)